src/confdb/confdb.c

```diff
@@ -1022,6 +1028,21 @@ static int confdb_get_domain_internal(struct confdb_ctx *cdb,
         }
     }
 
+    /* Read enabled attribute */
+    tmp = ldb_msg_find_attr_as_string(res->msgs[0], CONFDB_DOMAIN_ENABLED, NULL);
+    if (! tmp) {
+        domain->enabled = -1;
+    } else if (strcasecmp(tmp, "true") == 0 ) {
+        domain->enabled = 1;
+    } else if (strcasecmp(tmp, "false") == 0) {
+        domain->enabled = 0;
+    } else {
+        ret = EINVAL;
+        DEBUG(SSSDBG_FATAL_FAILURE,
+              "Invalid value for %s\n", CONFDB_DOMAIN_ENABLED);
+        goto done;
+    }
```

Comment:
Actually remove the whole code since domain->enabled is not used.

Reply:
Done